SU2_CFD/src/solvers/CMeshSolver.cpp

```diff
@@ -502,11 +502,12 @@ void CMeshSolver::DeformMesh(CGeometry **geometry, CNumerics **numerics, CConfig
   /*--- Clear residual (loses AD info), we do not want an incremental solution. ---*/
   SU2_OMP_PARALLEL {
     LinSysRes.SetValZero();
+    LinSysSol.SetValZero();
```

Comment:
Only if time_domain and FSI please, otherwise this is very inefficiency for steady problems where the linear system gets progressively cheaper to solve.

Reply:
Yes, updated